llvm/test/Transforms/SimplifyCFG/AMDGPU/switch-to-select.ll

```diff
@@ -0,0 +1,312 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
```

Comment:
Shouldn't there be a non-AMDGPU test case since this is a general optimization?

Reply:
Yes, I will do it that way.